samples/hssi/hssi_100g_cmd.h

```diff
@@ -228,6 +496,49 @@ class hssi_100g_cmd : public hssi_cmd
         enable_eth_loopback(eth_ifc, false);
     }
 
+    /* Monitor Implementation */
+
+    if (monitor_ != "none")
```

Comment:
if (continuous_ == on  && contmoitor_  != 0) {


}

Reply:
sure we will update it